### PR TITLE
[ADD] project_task_service_partner: Create date in res partner and pr…

### PR DESCRIPTION
**Pull request by Berezi**
…oject task. Patient in project and task.

### CODE REVIEW

**Comment by Berezi** [comment]
@alfredoavanzosc @anajuaristi @Daniel-CA @oihane review please.

File: project_task_service_partner/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    partner_shipping_id = fields.Many2one(
+        string='Patient', comodel_name='res.partner',
+        related='sale_order_id.partner_shipping_id')
```
Comment (anajuaristi) [inline project_task_service_partner/models/project_task.py]: related='project_id.partner_shipping_id')

File: project_task_service_partner/views/project_project_views.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="project_project_view_form" model="ir.ui.view">
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="sale_timesheet.project_project_view_form"/>
```
Comment (anajuaristi) [inline project_task_service_partner/views/project_project_views.xml]: ref="project_project_view_form"/>

File: project_task_service_partner/views/project_project_views.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <field name="arch" type="xml">
+            <field name="bill_type" position="after">
+                <field name="partner_shipping_id"/>
+            </field>
+        </field>
+    </record>
+    <record id="view_project" model="ir.ui.view">
```
Comment (anajuaristi) [inline project_task_service_partner/views/project_project_views.xml]: project_project_view_tree

File: project_task_service_partner/views/project_task_views.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_task_form2" model="ir.ui.view">
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2"/>
```
Comment (anajuaristi) [inline project_task_service_partner/views/project_task_views.xml:5]: project.view_task_form

File: project_task_service_partner/views/project_task_views.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        <field name="arch" type="xml">
+            <field name="project_privacy_visibility" position="before">
+                <field name="partner_shipping_id"/>
+            </field>
+        </field>
+    </record>
+    <record id="view_task_tree2" model="ir.ui.view">
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_tree2"/>
```
Comment (anajuaristi) [inline project_task_service_partner/views/project_task_views.xml:14]: project.view_task_tree

**Comment by Berezi** [comment]
@anajuaristi @oihane @Daniel-CA @alfredoavanzosc  review please.

**Review by oihane — CHANGES_REQUESTED** [review]
El rebase no es correcto, no se puede mergear como está

**Comment by Berezi** [comment]
@anajuaristi @oihane @Daniel-CA @alfredoavanzosc review please.

**Comment by oihane** [comment]
Este modulo tiene que ir a la rama project-addons

**Pull request closed without merging**